packages/react-components/src/components/CaptionsAndRTTAnnouncer.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import React, { useEffect, useState } from 'react';
+import { hiddenAnnouncementClassName } from './styles/Captions.style';
+import { CaptionsInformation, RealTimeTextInformation } from './CaptionsBanner';
+
+/** @internal */
+export type _CaptionsAndRTTAnnouncerProps = {
+  /**
+   * Array of captions to be displayed
+   */
+  captions: CaptionsInformation[];
+  /**
+   * Array of finalized and partial real time text messages
+   */
+  realTimeTexts?: {
```

Comment:
could we simplify this component since its just for announcements to only take in the strings it needs to announce? or is there some other processing that needs to be done before the announcement happens?